apps/explorer/src/ui/PlaceholderTable.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import TableCard from './TableCard';
+
+import placeholdertheme from './placeholder.module.css';
+
+type DataType = {
+    rowCount: number;
+    rowHeight: string;
+    colHeadings: string[];
+    colWidths: string[];
+};
+
+export function PlaceholderBox({
```

Comment:
I'd cut this out into a separate `ui/Placeholder` which is just this base component.

Reply:
Done